readyapi4j-groovy-dsl/src/test/groovy/com/smartbear/readyapi4j/dsl/pro/DataSourceTestStepDslTest.groovy

```diff
@@ -0,0 +1,88 @@
+package com.smartbear.readyapi4j.dsl.pro
+
+import com.smartbear.readyapi.client.model.DataSourceTestStep
+import com.smartbear.readyapi.client.model.ExcelDataSource
+import com.smartbear.readyapi.client.model.FileDataSource
+import com.smartbear.readyapi.client.model.RestTestRequestStep
+import com.smartbear.readyapi4j.TestRecipe
+import org.junit.Test
+
+import static com.smartbear.readyapi4j.dsl.ServerTestDsl.recipe
+
+class DataSourceTestStepDslTest {
+
+    private static final String EXCEL_FILE_PATH = 'C:/workspace/data-source-excel.xls'
+    private static final String FILE_PATH = 'C:/workspace/data-source.csv'
+    private static final String SHEET_NAME = 'Sheet1'
+    private static final String CELL = 'A1'
+    private static List<String> PROPERTIES = ['Name', 'Address']
+
+    @Test
+    void createsRecipeWithExcelDataSource() throws Exception {
+        TestRecipe testRecipe = recipe {
+            excelDataSource "ExcelDataSource", {
+                filePath EXCEL_FILE_PATH
```

Comment:
I think filePath should be the first argument in the "excelDataSource"/"usingExcelFile" method and name an optional attribute.